### PR TITLE
#1737 - Update institution location content (new)

### PR DESCRIPTION
**Pull request by hellolynn-tbtb**
### Updates

- Updated content on location form and summary
- Added banner on location edit
- Aligned tooltips styles

#### Location add/edit 
image

#### Location summary
image

#### Tooltips

Forms.io tooltips
image

Forms.io tooltips - custom (within content like NOA)
image

Vuetify tooltips
image

### CODE REVIEW

File: sources/packages/web/src/views/institution/EditInstitutionLocation.vue [hunk 1]
```diff
@@ -8,6 +8,17 @@
...
       />
     </template>
+    <template #alerts>
+      <banner
+        :type="BannerTypes.Info"
+        header="How to update information"
+        summary="Please email a request to update the location details. For the primary contact, you can update it without a request."
+      >
+        <template #actions>
+          <v-btn color="info">Email designat@gov.bc.ca</v-btn>
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/views/institution/EditInstitutionLocation.vue:18]: Can you please clarify what is the idea behind this button?
I believe that we are now using it at least in two places para it is actually not doing anything.

**Review by andrewsignori-aot — COMMENTED** [review]
Overall looks good and the tooltips look better across the different areas also. Just a small question, not directly related to this PR.

**Review by ann-aot — APPROVED** [review]
👍

File: sources/packages/web/src/views/institution/EditInstitutionLocation.vue (hunk 1 above)
Comment (andrewsignori-aot) [reply sources/packages/web/src/views/institution/EditInstitutionLocation.vue:18]: Thanks for the clarifications, Lynn.
Maybe we can have a clickable link to open the email client and a clipboard icon to copy it to the clipboard.

**Review by andrewsignori-aot — APPROVED** [review]
👍

**Pull request merged**